### PR TITLE
Added installation manifest for 3.5.0

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: mkdocs/docs/3.5.0/litmus-portal-crds-3.5.0.yml [hunk 1]
```diff
@@ -0,0 +1,3596 @@
...
+                                          additionalProperties:
+                                            type: string
+                                          description: Labels for the source pod
+                                          type: object
+                                        nodeSelector:
+                                          additionalProperties:
+                                            type: string
+                                          description: NodeSelector for the source pod
+                                          type: object
+                                        privileged:
```
Comment (ispeakc0de) [inline mkdocs/docs/3.5.0/litmus-portal-crds-3.5.0.yml:1209]: We can update crd to add tolerations

**Review by ispeakc0de — APPROVED** [review]

File: mkdocs/docs/3.5.0/litmus-3.5.0-without-resources.yaml [hunk 2]
```diff
@@ -0,0 +1,775 @@
...
+            - name: ARGO_WORKFLOW_CONTROLLER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/workflow-controller:v3.3.1"
+            - name: ARGO_WORKFLOW_EXECUTOR_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/argoexec:v3.3.1"
+            - name: LITMUS_CHAOS_OPERATOR_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-operator:3.3.0"
+            - name: LITMUS_CHAOS_RUNNER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-runner:3.3.0"
+            - name: LITMUS_CHAOS_EXPORTER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-exporter:3.3.0"
```
Comment (Jonsy13) [inline mkdocs/docs/3.5.0/litmus-3.5.0-without-resources.yaml:661]: Do we want to change them to 3.5.0?

File: mkdocs/docs/3.5.0/litmus-3.5.0-without-resources.yaml [hunk 3]
```diff
@@ -0,0 +1,775 @@
...
+            - name: CHAOS_CENTER_UI_ENDPOINT
+              value: ""
+            - name: INGRESS
+              value: "false"
+            - name: INGRESS_NAME
+              value: "litmus-ingress"
+            - name: CONTAINER_RUNTIME_EXECUTOR
+              value: "k8sapi"
+            - name: DEFAULT_HUB_BRANCH_NAME
+              value: "v3.4.x"
```
Comment (Jonsy13) [inline mkdocs/docs/3.5.0/litmus-3.5.0-without-resources.yaml:679]: Here too

File: mkdocs/docs/3.5.0/litmus-3.5.0-without-resources.yaml [hunk 4]
```diff
@@ -0,0 +1,775 @@
...
+            - name: LITMUS_AUTH_GRPC_ENDPOINT
+              value: "litmusportal-auth-server-service"
+            - name: LITMUS_AUTH_GRPC_PORT
+              value: ":3030"
+            - name: WORKFLOW_HELPER_IMAGE_VERSION
+              value: "3.4.0"
```
Comment (Jonsy13) [inline mkdocs/docs/3.5.0/litmus-3.5.0-without-resources.yaml:685]: Here too

File: mkdocs/docs/3.5.0/litmus-cluster-scope-3.5.0.yaml [hunk 5]
```diff
@@ -0,0 +1,802 @@
...
+            - name: ARGO_WORKFLOW_CONTROLLER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/workflow-controller:v3.3.1"
+            - name: ARGO_WORKFLOW_EXECUTOR_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/argoexec:v3.3.1"
+            - name: LITMUS_CHAOS_OPERATOR_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-operator:3.3.0"
+            - name: LITMUS_CHAOS_RUNNER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-runner:3.3.0"
+            - name: LITMUS_CHAOS_EXPORTER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-exporter:3.3.0"
```
Comment (Jonsy13) [inline mkdocs/docs/3.5.0/litmus-cluster-scope-3.5.0.yaml:670]: Same here

File: mkdocs/docs/3.5.0/litmus-namespaced-3.5.0.yaml [hunk 6]
```diff
@@ -0,0 +1,763 @@
...
+            - name: LITMUS_CHAOS_OPERATOR_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-operator:3.3.0"
+            - name: LITMUS_CHAOS_RUNNER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-runner:3.3.0"
+            - name: LITMUS_CHAOS_EXPORTER_IMAGE
+              value: "litmuschaos.docker.scarf.sh/litmuschaos/chaos-exporter:3.3.0"
+            - name: CONTAINER_RUNTIME_EXECUTOR
+              value: "k8sapi"
+            - name: DEFAULT_HUB_BRANCH_NAME
+              value: "v3.4.x"
```
Comment (Jonsy13) [inline mkdocs/docs/3.5.0/litmus-namespaced-3.5.0.yaml:648]: Same here

File: mkdocs/docs/3.5.0/litmus-namespaced-3.5.0.yaml [hunk 7]
```diff
@@ -0,0 +1,763 @@
...
+            - name: LITMUS_AUTH_GRPC_ENDPOINT
+              value: "litmusportal-auth-server-service"
+            - name: LITMUS_AUTH_GRPC_PORT
+              value: ":3030"
+            - name: WORKFLOW_HELPER_IMAGE_VERSION
+              value: "3.4.0"
```
Comment (Jonsy13) [inline mkdocs/docs/3.5.0/litmus-namespaced-3.5.0.yaml:654]: Same here

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**